snooty/test_postprocess.py

```diff
@@ -478,11 +478,11 @@ def test_toctree_self_add() -> None:
             """
 <toctree slug="/">
     <title><text>untitled</text></title>
-    <toctree slug="page1" drawer="True" />
-    <toctree slug="/" drawer="True">
+    <toctree slug="page1" drawer="True" tocicon="False" />
```

Comment:
If there's no tocicon set, we shouldn't have this entry in the options I think. Additionally, we should have a test *with* a tocicon. lmk if you need help with that!